### DISCUSSION TITLE
Github Account suspended without a suspension letter

### DISCUSSION THREAD

**Discussion by Leo-TX** [community/community#129340] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm not sure why my account was suspended, and I don't receive a suspension letter in my email. 

I have made a appeal to the Github Support, but I don't receive any response for few days.

I really want my account back cause I really need it. The account has a lot of my data and code.

My username is "TX-Leo". The ticket numberis 2848655.

**Comment by Shubham11440** [comment]
My account has been suspended without any reason. Could someone please help me regain access to my account? My ticket number is 2849590. Username - Shubham6621

**Comment by ebndev** [comment]
Hi @Leo-TX,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
